### DISCUSSION TITLE
Powerstream Energy Dashboard

### DISCUSSION THREAD

**Discussion by foxtrotoscar104** [tolwi/hassio-ecoflow-cloud#347] [Q&A]
Hi,

Is it possible use the PowerStream sensors in the energy dashboard for Solar production / battery? If I try to add into the dashboard I get a message saying no matching statistics found.

Thank you

Steve

**Comment by dopeytree** [comment]
I am just did this today.

- Go to settings -> Devices & Services -> Helpers
- Create new helper -> Integral
- Input sensor should be something like 'Powerstream Inverter Output Watts'
- Submit & check back in 30mins or so (when sunny) to make sure it's working. 
- It should take the watts & give a kWh figure (not sure if this is a total accumulation or per 24hrs)
- Might want to place with decimal points.
